templates/agent.kubelet.yaml

```diff
@@ -9,6 +9,11 @@ identities:
     parent_process: .*
     process: .*
   pushdown: []
+- node:
+    name: ecr_amazonaws
+    dns_pattern: .*ecr.*.amazonaws.com
```

Comment:
does it resolve to one (customer) account, or is a random number. If it is random and frequently changing in a single deployment, you want to use predicate_pushdown.